modules/openapi-generator/src/test/java/org/openapitools/codegen/OpenAPINormalizerTest.java

```diff
@@ -513,4 +513,23 @@ public void testSetContainerToNullable() {
         assertEquals(((Schema) schema4.getProperties().get("set_property")).getNullable(), true);
         assertEquals(((Schema) schema4.getProperties().get("map_property")).getNullable(), null);
     }
+
+    @Test
```

Comment:
why would it be different (in terms of what we are testing) than `testOpenAPINormalizerSimplifyOneOfAnyOf`?
and `testOpenAPINormalizerSimplifyOneOfAnyOfStringAndEnumString`

why not have same implementation, but different source spec?

Reply:
sure, not against it. can you file a PR to add those tests after this PR is merged?

Reply:
we will get it merged first and then create an issue to track better openapi spec 3.1 test